Service/src/main/java/Repository/ThreadModelRep/ThreadRepository.java

```diff
@@ -0,0 +1,48 @@
+package Repository.ThreadModelRep;
+
+
+import ThreadModel.Group;
+import ThreadModel.Mark;
+import ThreadModel.Salary;
+import ThreadModel.Theams;
+import Users.Trainer;
+import Users.UserImpl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+public interface ThreadRepository {
+    HashMap <Integer, Group> allGroup ();
+    HashMap <Integer, Theams> allTheams();
+    HashMap <Trainer, List<Salary>> trainerSalary ();
+    HashMap <UserImpl,HashMap<Theams, List<Mark>>> studentTheamMark (int StudentId);
```

Comment:
Хороший вариант.
Только пробельчики не забывай.
```suggestion
    HashMap <UserImpl, HashMap<Theams, List<Mark>>> studentTheamMark (int StudentId);
```

Reply:
Спасибо, океу.